collections/ansible_collections/zpe/nodegrid/plugins/action/setup.py

```diff
@@ -416,6 +445,7 @@ def run(self, task_vars=None):
 
             # Login
             conn_obj = pexpect.spawn(cmd, encoding="utf-8")
+            conn_obj.setecho(False)
```

Comment:
hey @zpe-leonardof  should this be turned on again later?

Reply:
No, it should be kept off. This action plugin also installs an SSH key and the key content can be exactly the same char sequence expected, breaking the logic.